content/docs/rfcs/58/README.md

```diff
@@ -0,0 +1,189 @@
+---
+slug: 58
+title: 58/RLN-V2
+name: Rate Limit Nullifier V2
+status: draft
+editor: Rasul Ibragimov <curryrasul@gmail.com>
+contributors: 
+- Lev Soukhanov <0xdeadfae@gmail.com>
+---
+
+# Abstract
+
+The `58/RLN-V2` protocol is improvement of `32/RLN-V1`, being more general construct, that allows to set various limits for an epoch (it's 1 message per epoch in `32/RLN-V1`) while remaining almost as simple as it predecessor. Moreover, it allows to set different rate-limits for different RLN app users based on some public data, e.g. stake or reputation.
```

Comment:
For RFC we recommend using semantic line break https://rfc.vac.dev/spec/1/#conventions

It makes git usage easier.